tests/testthat/test_mboost_surv_glmboost.R

```diff
@@ -1,15 +1,26 @@
 test_that("autotest ph", {
   learner = lrn("surv.glmboost", family = "coxph")
   expect_learner(learner)
+  exclude_str = "weights|utf8_feature_names|feat_single_factor|feat_all"
   # weights are fine for all predict types except 'distr'
-  result = run_autotest(learner, exclude = "weights|utf8_feature_names", check_replicable = FALSE)
```

Comment:
why do we now have to exclude more tests?

Reply:
The `importance` and `selected_features` functions don't work for datasets with factor features, I have a `stop` function there. But I added a extra test with such a dataset that has many factors